src/UglyToad.PdfPig/Geometry/PdfPoint.cs

```diff
@@ -82,6 +82,39 @@ internal PdfVector ToVector()
             return new PdfVector(X, Y);
         }
 
+        /// <summary>
+        /// Converts this <see cref="PdfPoint"/> into an array.
+        /// </summary>
+        /// <returns></returns>
+        public double[] ToDouble()
+        {
+            return new double[] { (double)this.X, (double)this.Y };
+        }
+
+        /// <summary>
+        /// Returns a value indicating whether this <see cref="PdfPoint"/> is equal to a specified <see cref="PdfPoint"/> .
+        /// </summary>
+        /// <param name="obj"></param>
+        /// <returns></returns>
+        public override bool Equals(object obj)
+        {
+            if (obj is PdfPoint)
```

Comment:
There's a slightly terser syntax for this:

```suggestion
            if (obj is PdfPoint point)
```

Then you can remove line 103, but I don't mind, I just know Resharper is going to nag me 😆

Reply:
Very good, thx!